src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const form = document.querySelector('.contacts__form');
```

Comment:
```suggestion
const form = document.querySelector('.contact-us__form');
```
fix selector for the form
<img width="1091" alt="image" src="https://github.com/mate-academy/layout_dia/assets/102411130/3869f4a4-1114-4b66-b834-c43cedb46441">